modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -1,14 +1,39 @@
 #include "opencv_coins.hpp"
 
 using namespace cv;
+using namespace std;
```

Comment:
I don't recommend to use `using namespace std;` - it may lead to some problems sometimes

Reply:
> I don't recommend to use `using namespace std;` - it may lead to some problems sometimes

Thank you, this problem is fixed